### DISCUSSION TITLE
"The Big Bang Theory Is True. Deal With It."

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#148] [Metaphysics and Philosophy]
Some lighter reading, written by John Horgan.

[https://johnhorgan.org/cross-check/the-big-bang-theory-is-true-deal-with-it](https://johnhorgan.org/cross-check/the-big-bang-theory-is-true-deal-with-it)

> "If you believe science actually discovers reality, and achieves truth, you must consider the possibility that the era of truly profound, evolutionary insights into nature is over."

$\mbox{ }$

Well then, the era of revolutionary insights into nature is not over!

**Comment by bgreenman** [comment]
The problem is its hard to find an alternative cosmological alternative, dosen't mean the Big Bang origin is correct, to find alternatives and the underlying simplicity the universe will require more advanced and sensitive instruments

**Reply by budrap00** [reply]
The problem isn't a lack of data,. The problem is that cosmologists have spent the last 100 years fitting all the new and unpredicted data into the expanding universe paradigm based on the simplistic assumptions of the FLRW model. The expanding universe is dogma not science, but to be an accepted member of the academic cosmologists cult you have to bow down to that sacred calf.

It is not information that is lacking but people who know how to think instead of just regurgitate what they were taught to think. There's plenty of knowledge on which to build a better cosmological model but it's not going to happen as long as the expanding universe is held as an inviolable belief.

**Reply by bgreenman** [reply]
The main problem is scientific ideological rigidity

**Comment by HanDeBruijn** [comment]
Quote from the article:
>Since that 1990 meeting, cosmologists have made one genuinely surprising discovery, that _the expansion of the universe is speeding up_.

From Wikipedia's [Scale factor (cosmology)](https://en.wikipedia.org/wiki/Scale_factor_(cosmology)) we have that
>The Hubble parameter is defined as:

$$
H(t) \equiv \frac{\dot{a}(t)}{a(t)}
$$

It is for sure that we must consider the [Dark-energy-dominated era](https://en.wikipedia.org/wiki/Scale_factor_(cosmology)#Dark-energy-dominated_era), because
>the **dark-energy-dominated era** is proposed as the last of the three phases of the known universe [ .. ]

**There are no epochs and eras in an Eternal Static Euclidean Universe.** Therefore only the "last" one might be relevant.
>For a dark-energy-dominated universe, the evolution of the scale factor [ .. ]

is given in our own notation by

$$
a(t) = a_0e^{H(t-t_0)}
$$

with $H=H_0$ as a constant. By differentiation to $t$ it follows that

$$
\frac{\dot{a}}{a} = H_0 \quad \mbox{and} \quad \frac{\ddot{a}}{a}= H_0^2
$$

The latter is indeed a speeding up of the expansion. This is all in concordance with the Variable Mass Theory. Because in [Length Contraction](http://www.alternatievewiskunde.nl/cosmology/lengte.htm) it is argued that
>Length _Contraction_ is observationally equivalent with the _Expansion_ of Empty Space. 

So the latter is the reverse of the former. Formally:

$$
\frac{L_0}{L(t)} = e^{H(t-t_0)} \quad \Longrightarrow \quad \frac{a(t)}{a_0} = e^{H(t-t_0)}
$$

Which is exactly the same formula as with the _dark-energy-dominated era_ in mainstream theory. The connection between mass density and the Hubble parameter is a bit different in our theory, but that's of minor importance now.

$$
H_0 = \sqrt{w.8\pi G.\rho_{full}} \quad \mbox{with} \quad w = 0.3883945571 ....
$$

Wikipedia's [Accelerating expansion of the universe](https://en.wikipedia.org/wiki/Accelerating_expansion_of_the_universe) tells us another story with standard cosmology, though. First we assume, of course, that there is NO _curvature of the universe_, hence $\kappa=0$. Then we have the simplified formula

$$
H^2 = \left(\frac{\dot{a}}{a}\right)^2 = \frac{8\pi G}{3}\rho
$$

The so-called acceleration equation is aimed to describe the evolution of the scale factor with time:

$$
\frac{\ddot{a}}{a} = - \frac{4\pi G}{3}\left(\rho + \frac{3P}{c^2}\right) \ne H_0^2
$$

Okay, I can be forgiving for the factor $4\pi/3$ and the term $3P/c^2$. _But I don't comprehend **the minus sign**_.
How can there be an acceleration with a minus sign? I thought that must result in a _slowing down instead of a speeding up_.

**Reply by bgreenman** [reply]
My view is that we don't know the full composition of empty space-time

**Reply by RedshiftDrift** [reply]
@HanDeBruijn the [pressure $P$ is defined as being "negative when repulsive"](https://en.wikipedia.org/wiki/Accelerating_expansion_of_the_universe#Explanatory_models).  It is given by the equation $P(t) = w c^2 \rho(t)$ where [in ΛCDM](https://en.wikipedia.org/wiki/Lambda-CDM_model#Parameters) $w = -1.03 \pm 0.03$ (different meaning than your $w \simeq 0.39$).

Then the minus sign goes away

$$ \frac{\ddot{a}(t)}{a(t)} = -\frac{4\pi G}{3}\left(\rho(t) +\frac{3P(t)}{c^2}\right) = -\frac{4\pi G}{3}\big(\rho(t) -3\rho(t)\big) = \frac{8\pi G}{3}\rho(t) = H(t)^2 $$

**Reply by bgreenman** [reply]
According to wikipedia the minus sign is removed when anti-matter or massive gravity contradictory to General Relativity occurs

**Reply by HanDeBruijn** [reply]
@RedshiftDrift 

>It is given by the equation $P(t) = w c^2 \rho(t)$ where in &Lambda;CDM $w = -1.03 \pm 0.03$ (different meaning than your $w \simeq 0.39$).

That opens a new perspective ! The two $w$ variables are obviously related. Let's call them respectively $w_{bb}$ (Big Bang) and $w_{vm}$ (Variable Mass). Then we have

$$
H_0^2 = 2w_{vm} . 4\pi G \rho = - 4\pi G \rho \left(\frac{1}{3} + w_{bb}\right)
$$

More about pressure and density of the medium that does not exist is found in [Equation of state (cosmology)](https://en.wikipedia.org/wiki/Equation_of_state_(cosmology)#Acceleration_of_cosmic_inflation). There we read:

>Using the existing data, it is still impossible to distinguish between phantom $w \lt -1$ and non-phantom $w \geq -1$.

$$
w_{bb} = - \frac{1}{3} - 2.w_{vm} \simeq -1.11 \lt -1
$$

Come on ! It's **phantom** of course.

**Reply by bgreenman** [reply]
Correct its phantom, scaler fields are useful

**Reply by RedshiftDrift** [reply]
> Come on ! It's phantom of course.🫥

I have objections to equating scalars that come from different physical models, as in

$$
2w_{vm} . 4\pi G \rho \ {^?\overset{?}{=}}{^?} \ - 4\pi G \rho \left(\frac{1}{3} + w_{bb}\right)
$$

It is like comparing $apples \land oranges$.

In any case, we'll keep this in mind when $w_{bb}$ is known to a higher accuracy! 😉

**Reply by bgreenman** [reply]
Correct

**Comment by RedshiftDrift** [comment]
Eric Lerner [posted a comment](https://johnhorgan.org/cross-check/the-big-bang-theory-is-true-deal-with-it), and John Horgan replied... but remains skeptical!

**Comment by bgreenman** [comment]
The John Horgan debate is interesting, but we still don't know many things about physics and astrophysics basics

**Comment by OlliSant** [comment]
BB is wrong because the universe does not begin and expand. It is eternal and infinite. Infinite, but not absolutely infinite. We know this by observation and rationally, together both ways. 

The local part-universe begins. And does not expand.

All this is scientific knowledge. We are too timid, shy. The observable universe is not a real entity of the reality, it is just some part of the universe. Is bigger and bigger with every new telescope. We can already estimate the rest. It is similar: galaxies.

**Comment by Francois-Zinserling** [comment]
It only gets worse ...

Now there was also a ..
[Dark Big Bang](https://www.economist.com/science-and-technology/2024/03/06/physicists-are-reimagining-dark-matter)

**Reply by HanDeBruijn** [reply]
More references: [There May Have Been a Dark Big Bang](https://www.popularmechanics.com/space/deep-space/a43096887/dark-matter-big-bang/) , [A second big bang?](https://www.newscientist.com/article/mg26034640-300-a-second-big-bang-the-radical-idea-rewriting-dark-matters-origins/) , [Dark Matter and Gravity Waves from a Dark Big Bang](https://arxiv.org/abs/2302.11579) , [The universe may have started with a dark Big Bang](https://phys.org/news/2023-03-universe-dark-big.html) , [There may have been a second Big Bang, new research suggests](https://www.livescience.com/a-dark-big-bang-may-have-flooded-the-universe-with-invisible-matter-new-study-proposes) ... and exploding ...

**Reply by HanDeBruijn** [reply]
It's so simple with the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs). Because there are only two phenomena here that need an alternative explanation: Dark Matter and Dark Energy.
### Dark Matter
Key reference is a comment in [Checking in on Troubles with Dark Matter](https://github.com/orgs/a-cosmology-group/discussions/80#discussioncomment-6328995) . It is proved that (my version of) the Variable Mass Theory explains the flat rotation curves of galaxies without the need to resort to the existence of dark matter. MOND is a (proper) subset of this theory.
### Dark Energy
Google says _Think of dark energy as the "evil counterpart" to gravity – an "anti-gravity" force providing a negative pressure that fills the universe and stretches the very fabric of spacetime._ And that's all? A [comment](https://github.com/orgs/a-cosmology-group/discussions/148#discussioncomment-7047729) and a [reply](https://github.com/orgs/a-cosmology-group/discussions/148#discussioncomment-7068915) in this thread already lead to an explanation.

@RedshiftDrift writes:
>I have objections to equating scalars that come from different physical models

Dismissed. The expanding space in Big Bang Theory is mathematically equivalent with a Static Euclidean Variable Mass Universe.

**Reply by RedshiftDrift** [reply]
In cosmology, pigs already fly. So we need big umbrellas like "expanding space", "dark big bang"...
